index.html

```diff
@@ -267,43 +226,53 @@ <h3 class="sidebar_list_item_head">
-                            obtaining information on any current or future job openings, and for inclusion into 
-                            a mailing list that will be used in future communications.</p>
-                        <p>Information collected will be treated in accordance with the System of Records 
-                            Notice (“SORN”), CFPB.009 Employee Administrative Records, 78 FR 67340. This 
-                            information will not be disclosed as outlined in the Routine Uses for the SORN. 
-                            Direct identifying information will only be used to facilitate the potential 
-                            job openings, and to provide you with tailored updates from Human Resources, 
-                            and will be kept private except as required by law.</p>
-                        <p>This collection of information is authorized by Pub. L. No. 111-203, Title X, 
-                            Sections 1013 and 1022, codified at 12 U.S.C. §§ 5493 and 5512.</p>
-                        <p>Submission of your resume is voluntary. You are not required to submit or 
-                            provide any identifying information; however, not doing so may result in the CFPB 
-                            being unable to contact you about potential employment opportunities.</p>
-                    </div>
+                <div class="job-half">
+                    <p>
+                        Our mission demands smart, usable, and reliable technology.
+                        Come serve consumers by helping us design and build it.
+                    </p><p>
+                        Learn more about the work we do at our page for the
+                        <a href="http://www.consumerfinance.gov/jobs/technology-innovation-fellows/">Technology &amp; Innovation Fellowship</a><i class="icon-external-link"></i>.
+                    </p><p>
+                        Share your contact information and
+                        we'll contact you with software development opportunities throughout the year.
+                    </p>
+                </div>
+                <div class="job-half">
+                    <form class="signup-form">
+
+                        <label class="signup_fullname">
+                            Full Name
+                            <input type="text" class="js-recruit-name" aria-label="Full Name">
+                        </label>
+                        <label class="signup_email">
+                            Email
+                            <input type="email" class="js-recruit-email" aria-label="Email">
+                        </label>
+                        <label>
+                            <span>Select a discipline that matches your skillset the most</span>
+                            <select class="js-recruit-discipline" aria-label="Discipline">
```

Comment:
I can't seem to find anything that says anything about that. Why is that the case?